Dispatchfile

```diff
@@ -100,6 +100,6 @@ dind_task("test-helm3",
 )
 
 action(tasks=["test-helm2","test-helm3"], on=pullRequest())
-action(tasks=["publish"], on=push(branches=["master"]))
+action(tasks=["publish"], on=push(branches=["release/kommander-0.8.x"]))
```

Comment:
oh boy - good catch!

this shouldnt be hardcoded in the first place. is there any way to get the target branch from env vars? cc @faiq any idea?